app_changelog.go

```diff
@@ -0,0 +1,12 @@
+package versionbundle
+
+type AppsLog map[string][]Log
+
+type Log struct {
+	Description string   `json:"description"`
+	Type        string   `json:"type"`
+	Urls        []string `json:"urls"`
```

Comment:
```suggestion
	URLs        []string `json:"urls"`
```

Casing should be the same as the existing type.

Reply:
This had been deleted since we decided to use `Changelog`.